Game/Controllers/MouseController.cs

```diff
@@ -29,6 +29,30 @@ public MouseController(Game game, IPlayer player)
         public void Update()
         {
             MouseState mouseState = Mouse.GetState();
+            ICommand command;
+            if (mouseState.LeftButton == ButtonState.Pressed)
+            {
+                if (mouseState.X < 2 * Constants.BlockSize)
+                {
+                    command = new ChangeRoomWestCommand(game);
+                    command.Execute();
+                }
+                if (mouseState.X > 14 * Constants.BlockSize)
+                {
+                    command = new ChangeRoomEastCommand(game);
+                    command.Execute();
+                }
+                if (mouseState.Y < 2 * Constants.BlockSize + Constants.HudAndMenuHeight)
+                {
+                    command = new ChangeRoomNorthCommand(game);
+                    command.Execute();
+                }
+                if (mouseState.Y > 9 * Constants.BlockSize + Constants.HudAndMenuHeight)
+                {
+                    command = new ChangeRoomSouthCommand(game);
+                    command.Execute();
+                }
```

Comment:
They do not unexecute, not that there's anything in the unexecute section. Good practice they should unexecute I guess? Just didn't bother right now.

Reply:
That's fine.